aws/awserr/types.go

```diff
@@ -2,6 +2,16 @@ package awserr
 
 import "fmt"
 
+const (
+	// CodeSerialization is the serialization error code that is received
+	// during protocol unmarshaling.
+	CodeSerialization = "SerializationError"
```

Comment:
Current the `awserr` package has no errors codes in it. All current request related error codes are defined in the `request` package. 

Error codes in `request` package were miss-named and not caught before release of the feature :(